packages/rum-core/src/domain/vital/vitalCollection.spec.ts

```diff
@@ -30,78 +32,112 @@ describe('vitalCollection', () => {
   })
 
   describe('custom duration', () => {
-    describe('createVitalInstance', () => {
-      it('should create duration vital from a vital instance', () => {
+    describe('startDurationVital', () => {
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏You could add a test explaining what happens when creating multiple vitals with the same name and using stop with the name.

Reply:
Done